client/src/components/forms/MailingList/slides/Social.tsx

```diff
@@ -23,54 +24,131 @@ const styles = (theme: Theme) =>
       flexDirection: 'column',
       marginBottom: theme.spacing(6),
     },
+    centerText: {
+      textAlign: 'center',
+    },
     iconsContainer: {
       display: 'flex',
       justifyContent: 'space-around',
       width: '100%',
+      flexWrap: 'wrap',
+      marginBottom: theme.spacing(2),
+      [theme.breakpoints.down('sm')]: {
+        marginBottom: 0,
+      },
     },
     icon: {
       width: ICON_SIZE,
       height: ICON_SIZE,
     },
+    button: {
+      [theme.breakpoints.down('sm')]: {
+        marginBottom: theme.spacing(1),
+      },
+    },
+    scContainer: {
+      cursor: 'pointer',
+    },
+    sc: {
+      maxWidth: '100%',
+      maxHeight: '100%',
+    },
+    snapCodeWrapper: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    snapCodeContainer: {
+      height: 400,
+      [theme.breakpoints.down('sm')]: {
+        height: 'unset',
+        width: `calc(100% - ${2 * theme.spacing(2)}px)`,
+        margin: `0 ${theme.spacing(2)}px`,
+        marginTop: theme.spacing(2),
+      },
+    },
+    snapCode: {
+      maxHeight: '100%',
+      [theme.breakpoints.down('sm')]: {
+        maxHeight: 'unset',
+        maxWidth: '100%',
+      },
+    },
   });
 
 function Social({ classes }: WithStyles<typeof styles>) {
+  const [isSCHidden, setIsSCHidden] = useState(true);
+
   return (
     <div className={classes.container}>
       <div className={classes.textContainer}>
-        <Typography color="textPrimary" variant="h4">
+        <Typography
+          color="textPrimary"
+          variant="h4"
+          className={classes.centerText}
+        >
           Thanks for your feedback!
         </Typography>
-        <Typography color="textPrimary" variant="body1">
-          {"You're now a part of our mailing list"}
-        </Typography>
-        <Typography color="textPrimary" variant="body1">
-          Make sure to follow us on social media
+        <Typography
+          color="textPrimary"
+          variant="body1"
+          className={classes.centerText}
+        >
+          Make sure to follow us on social media:
         </Typography>
       </div>
       <div className={classes.iconsContainer}>
         <Button
+          className={classes.button}
           onClick={() =>
             window.open('https://www.facebook.com/uWaterlooACS', '_blank')
           }
         >
           <FacebookIcon className={classes.icon} />
+          facebook.com/uWaterlooACS
         </Button>
```

Comment:
Button adds more styling (hover states and alignment stuff)

Reply:
Ohh makes sense. What about wrapping the button in an `a` tag?

Reply:
Ohh I see what you're saying. Do you want to use an anchor so we can specify target and all that?
I could try it

Reply:
Mostly just because this is really a link and an anchor is most appropriate. I think using an anchor in these cases is good for accessibility and SEO.

I'm tripping tho because having a `button` in an `a` isn't valid HTML.

Reply:
Oh apparently we can add `href` and `target` properties to the button itself and Material UI will use an anchor element. [docs 💸](https://material-ui.com/api/button/#props)